include/image-commands.mk

```diff
@@ -226,6 +226,11 @@ define Build/copy-file
 	cat "$(1)" > "$@"
 endef
 
+define Build/dlink-sge-image
+	$(STAGING_DIR_HOST)/bin/dlink-sge-image $(1) $@ $@.enc
```

Comment:
This was merged into master in 4c0dc68f466, please remove it here.

Reply:
fixed